back-end/app.js

```diff
@@ -43,24 +68,45 @@ app.get('/tv_users/:id', (req, res, next) => {
       res.json(response.data)
     })
     .catch((err) => {
-      next(err)
+      if (err.response.status === 500) {
+        if (req.params.id in mockUserAPI) {
+          res.status(200).json(mockUserAPI[req.params.id])
+        } else {
+          res.status(400).json('user with requested id not found')
+        }
+      } else {
+        next(err)
+      }
     })
 })
 
 app.post('/tv_users', (req, res, next) => {
-    axios.post(`https://my.api.mockaroo.com/tv_users.json?key=${process.env.API_KEY_MOCKAROO}&__method=POST`, {
-        "username": req.body.username,
-        "email": req.body.email,
-        "password": req.body.password
+  axios
```

Comment:
looks good! just a small question - why removing api key

Reply:
eek, thanks for pointing that out